app/src/constants/UrlLink.js

```diff
@@ -0,0 +1 @@
+export const BASE_API_URL = 'http://localhost:8080/api/'
```

Comment:
Can you move this to a property file. 
local and production should have diff property files.